docs/programs/examples.md

```diff
@@ -1,7 +1,132 @@
 ---
+date: Apr 26, 2024
 title: "Program Examples"
+description:
+  "A list of Solana program examples in different languages and frameworks,
+  which can help you learn and use as reference for your own projects."
+tags:
+  - quickstart
+  - program
+  - anchor
+  - javascript
+  - native
+  - rust
+  - solang
+  - token22
+  - token extensions
+keywords:
+  - rust
+  - cargo
+  - toml
+  - program
+  - tutorial
+  - intro to solana development
+  - blockchain developer
+  - blockchain tutorial
+  - web3 developer
+  - anchor
+  - solang
 ---
 
+The
+"[Solana Program Examples](https://github.com/solana-developers/program-examples)"
+repository on GitHub offers several subfolders, each containing code examples
+for different Solana programming paradigms and languages, designed to help
+developers learn and experiment with Solana blockchain development.
+
+You can find the examples in the `solana-developers/program-examples` together
+with README files that explain you how to run the different examples. Most
+examples are self-contained and are available in Native rust,
+[Anchor](https://www.anchor-lang.com/docs/installation),
+[Seahorse](https://seahorse-lang.org/) and it also contains a list of examples
```

Comment:
Just a suggestion that since Seahorse is looking for a permanent maintainer, we hold off to avoid steering people in this direction. We can refocus on Seahorse once we have a maintainer and it's up to date with Solana CLI releases.

Reply:
Anvit took over being the maintainer a few months ago. Is he not actually maintaining it?